README.md

```diff
@@ -85,6 +85,7 @@ Kinvey CLI is distributed as an NPM package. After you install NPM, run the foll
 * `org apply`
 
     Applies org configuration file to the specified org or to the active one.
+    Also applies organization services and organization apps configuration files if they are in the respective `services` and `applications` subfolders.
```

Comment:
An org config file contains org-level services and apps, so applying an org file means modifying these services and apps. It is what the org config file is all about. 
I think that if the org config file needs clarifications they should be in the `Configuration management` section. The `Commands` section is just for command-specific info but you should know what is config management and this is the place to find how to apply the org config file for example. At least, this is the way I see it.

Edit: The same comment applies to the other modifications in the `Commands` section.